### PR TITLE
AddingPowerSupport_aaargh

### PR DESCRIPTION
**Pull request by santosh653**
creating aaargh.travis.yml with proposed changes.,

### CODE REVIEW

File: travis-ymls/aaargh.travis.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# ----------------------------------------------------------------------------
+#
+# Package             : pyarrow 
+# Source Repo         : https://github.com/apache/arrow
```
Comment (seth-priya) [inline travis-ymls/aaargh.travis.yml:4]: The commit and the file name suggest that this for package "aargh", but the contents seem to be for "pyaarow", please check and update.

**Review by seth-priya — CHANGES_REQUESTED** [review]

**Comment by seth-priya** [comment]
@santosh653 please squash the commits

**Comment by santosh653** [comment]
@seth-priya can you please check now?

**Comment by seth-priya** [comment]
@santosh653 I still see 4 commits

**Comment by santosh653** [comment]
I am closing this pull request.

**Pull request closed without merging**